src/ui/ImageListModel.java

```diff
@@ -0,0 +1,44 @@
+package ui;
+import javax.swing.*;
+import java.io.File;
+
+/**
+ * THis class are in charge of create and send the list model
+ * and will be responsible load the images that user choose
+ */
+public class ImageListModel {
+    ImageIconToCharge imageIconToCharge;
+    DefaultListModel listIMag = new DefaultListModel();
+
+    /**
+     * this method is in charge of filling the list model which ImageIcons
+     * which will return to the LeftPanel class for her to be loaded
+     * @return returns the  variable  listModel filled whit imagesIcon
+     */
+    public DefaultListModel listModel(String path) {
+        try {
+            File folder = new File(path);
+            File[] listOfFiles = folder.listFiles();
+            int index = 0;
+            for (int i = 0; i < listOfFiles.length; i++) {
+                String name = listOfFiles[i].toString();
+                if (name.endsWith("jpg") || name.endsWith("bmp") || name.endsWith("png")) {
```

Comment:
Bruno has already a method to validate the file extensions, you should use that one